docs/en/operations/system-tables/codecs.md

```diff
@@ -0,0 +1,44 @@
+---
+description: 'System table containing information about codecs
+  in queue.'
+keywords: ['system table', 'codecs', 'compression']
+slug: /operations/system-tables/codecs
+title: 'system.codecs'
+---
+
+Contains information about compression and encryption codecs.
+
+You can use this table to get information about the available compression and encryption codecs
+
+The `system.codecs` table contains the following columns (the column type is shown in brackets):
+
+- `name` ([String](../../sql-reference/data-types/string.md)) — Codec name.
+- `is_compression` ([UInt8](/sql-reference/data-types/int-uint#integer-ranges)) — The codec performs data compression.
```

Comment:
Users will not understand the distinction between `is_compression` and `is_generic_compression`. Two options:
- either omit one of the fields, or
- make the description ("The codec performs data compression", ...) more illustrative.